src/Tools/CI/ClearIISFolders.ps1

```diff
@@ -1,6 +1,6 @@
 param([string]$rootIisFolder,[string]$folder)
 $folders = ("$rootIisFolder\dotvvm.owin.api","$rootIisFolder\dotvvm.core.api", "$rootIisFolder\dotvvm.$folder")
 
-foreach($f in $folders){ if($f){ new-item -path $f -name app_offline.htm -itemtype file -value "Deplyoment"}}  
+foreach($f in $folders){ if($f){ new-item -path $f -name app_offline.htm -itemtype file -value "Deployment"}}  
```

Comment:
I am not really familiar with these scripts. Is this change fine or can it break anything? RFC @quigamdev @tomasherceg

Reply:
This is ok.